assignments/array-methods.js

```diff
@@ -54,30 +54,61 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 {"id":50,"first_name":"Shell","last_name":"Baine","email":"sbaine1d@intel.com","shirt_size":"M","company_name":"Gabtype","donation":171}];
 
 // ==== Challenge 1: Use .forEach() ====
-// The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
-let fullName = [];
+// The event director needs both the first and last names of each runner for their running bibs.
+//Combine both the first and last names into a new array called fullName. 
+let fullName = []
+runners.forEach(function(currentValue) {
```

Comment:
runner would be a more descriptive than currentValue (nice use of camel case though!!)